packages/cli/src/lingui-compile.ts

```diff
@@ -84,8 +84,8 @@ function command(config, options) {
         const typescriptPath = compiledPath.replace(/\.jsx?$/, "") + ".d.ts"
         fs.writeFileSync(
           typescriptPath,
-          `import { AllMessages } from '@lingui/core';
-declare const messages: AllMessages;
+          `import { Messages } from '@lingui/core';
+declare const messages: Messages;
 export = messages;
```

Comment:
```suggestion
export { messages };
```

I believe we should also change this, because messages are no longer default export, but a named one.

Reply:
Ah yup